editor/editor_folding.cpp

```diff
@@ -173,6 +173,9 @@ void EditorFolding::load_scene_folding(Node *p_scene, const String &p_path) {
 	for (int i = 0; i < unfolds.size(); i += 2) {
 		NodePath path = unfolds[i];
 		PoolVector<String> un = unfolds[i + 1];
+		if (p_scene->has_node(path) == false) {
```

Comment:
You can use `if (!p_scene->has_node(path))` instead.

But it would maybe be better to make `Node::_get_node` public and use that directly, and expose it to scripting languages as `Node.get_node_or_null`, see #24779.

Reply:
Currently busy, but I'll look into get_node_or_null. The function just needs to be exposed?

Reply:
Updated.

Reply:
Please review.